lib/saml/user.rb

```diff
@@ -58,7 +58,7 @@ def to_hash
     private
 
     def serializable_attributes
-      %i[authn_context]
+      %i[authn_context has_va_sso]
```

Comment:
Does it make sense to instead include `issuer` as a serializable attribute as opposed to `has_va_sso`, that is, serailize the "data" and not the "logic on that data"? 

I could be misunderstanding how these `serializable_attributes` are used.

Reply:
there was a commit that tried that, https://github.com/department-of-veterans-affairs/vets-api/pull/3933/commits/27936b74cc8245d99688b91ceb5f789bb36358cb, however the thought was we we're just seperating the "eauth.va.gov" domain matching into 2 different places and if that regex ever needed to be changed it was more likely to cause problems having the logic in multiple modules.

Reply:
Ah, glad I wasn't being crazy. That makes sense. 👍